modules/analysis.nf

```diff
@@ -65,9 +63,7 @@ process nextclade {
     """
     nextclade --input-fasta ${fasta} \
 	--input-root-seq ${reffasta} \
-	--input-gene-map=nextclade_files/genemap.gff \
-	--input-tree=nextclade_files/tree.json \
-	--input-qc-config=nextclade_files/qc.json \
+	--input-dataset=nextclade_files \
```

Comment:
I do not intend to change this. Should I revert back to the previous version?

Reply:
This was introduced by Nick as part of 347803c